orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -2805,53 +2801,20 @@ Return true if all elements are true and false otherwise.
               /*is_homogeneous*/ false,
               /*min_arity*/ 1)
       .Attr("name", "Name of ATen operator.", AttributeProto::STRING)
-      .Attr("custom_attributes_json", "custom attributes in JSON format.", AttributeProto::STRING, false)
+      .Attr("overload_name", "Overload name of ATen operator.", AttributeProto::STRING, false)
+      .Attr("output_types", "Output types of ATenOp's gradient op.", AttributeProto::INTS, false)
```

Comment:
is "output_types" specific to gradient op only? 
How do we set output type for ATen forward ops?

Reply:
This is the issue I mentioned before, that we need to add some code in symbolic_shape_infer.py to support this. Ideally we need exporter to help, by setting some information in the node attribute to help us.